cluster.go

```diff
@@ -1018,6 +1088,8 @@ func (c *clusterClient) Nodes() map[string]Client {
 }
 
 func (c *clusterClient) Close() {
+	close(c.refreshStop)
```

Comment:
Use the `c.stop` atomic flag to avoid closing the channel more than once.

Reply:
[1181ffb](https://github.com/redis/rueidis/pull/638/commits/1181ffbe8041a47911e438c99bb2464e46a6d00e)